akka-http-tests/src/test/scala/akka/http/javadsl/server/HttpAppSpec.scala

```diff
@@ -147,18 +145,18 @@ class HttpAppSpec extends AkkaSpec with RequestBuilding with Eventually {
 
     "notify" when {
 
-      "shutting down" in withSneaky { (sneaky, host, port) ⇒
+      "shutting down" in withSneaky { sneaky ⇒
 
         val server = Future {
-          sneaky.startServer(host, port, ServerSettings.create(ConfigFactory.load))
+          sneaky.startServer("localhost", 0, ServerSettings.create(ConfigFactory.load))
```

Comment:
I remember perhaps @patriknw mentioning that binding to localhost (or was it 127.0.0.1) did not work for him sometimes...? Do we have a preference where we should bind to? We use either in this PR

Reply:
I used `127.0.0.1` where it is later checked and `localhost` otherwise. The previous version also used `localhost` through the default of `temporaryServerHostnameAndPort` so hopefully it will be fine.

Reply:
Okey cool, thanks for checking!